### DISCUSSION TITLE
About `legacy pre-eip-155 transactions not supported` warning

### DISCUSSION THREAD

**Discussion by Ayuilos** [ProjectOpenSea/seaport#638] [Q&A]
This is a slightly off-topic issue sorry because I don't know a better way to contact your dev team efficiently.
I wonder under the most mainstream RPC nodes currently do not support EIP-155 disabled transactions situation, how did you send the pre-signed transaction as described in
https://github.com/ProjectOpenSea/seaport/blob/main/docs/Deployment.md?

**Comment by ryanio** [comment]
You would have to find a node that has that option enabled

**Reply by Ayuilos** [reply]
Yes ofc, so what's your option? You find a node or you open your own full node? If you have found a public node service could you please tell me?

**Reply by pamungkaski** [reply]
I usually tried one by one of the node list on https://chainlist.org/

**Reply by Ayuilos** [reply]
Thanks, I'll try!
